portfolio/src/main/webapp/style.css

```diff
@@ -6,4 +6,16 @@
 
 input { 
   display: block; 
-} 
\ No newline at end of file
+} 
+
+form {
```

Comment:
This works, especially as an exercise, but for a larger app I would question how sustainable it is to have javascript hide and sets of elements depending on login status.

Your users would not even know why they should log in (as the wouldn't know about the form which magically appears after they do is hidden).

You might want to consider how you'd break up your up into several html pages some of which might be redirecting to your login page.